src/test/resources/com/urswolfer/gerrit/client/rest/http/changes/reviewers.json

```diff
@@ -0,0 +1,21 @@
+)]}'
+[
+    {
+        "approvals": {
+            "Verified": "+1",
+            "Code-Review": "+2"
+        },
+        "_account_id": 1000096,
+        "name": "John Doe",
+        "email": "john.doe@example.com"
+    },
+    {
+        "approvals": {
+        "Verified": " 0",
```

Comment:
Please fix indention and add a "custom" label (like "My-Own-Label") when you have migrated to a Map.